### PR TITLE
rubocop: use new cops and take autocorrect for violations

### PR DESCRIPTION
**Pull request by ndushay**

### CODE REVIEW

File: modsulator.gemspec [hunk 1]
```diff
@@ -16,10 +16,13 @@ Gem::Specification.new do |s|
   s.platform    =  Gem::Platform::RUBY
   s.executables << 'modsulator'
 
+  s.required_ruby_version = '~> 3.0'
```
Comment (jcoyne) [inline modsulator.gemspec:19]: We haven't pushed this gem for 3 years. Does it still need to be a gem?

Comment (ndushay) [reply modsulator.gemspec:19]: Dunno.  Arcadia or someone(s) in metadata use it and I am not sure how they are using it.

Comment (jcoyne) [reply modsulator.gemspec:19]: It sounds like they are not using it as a gem.

Comment (ndushay) [reply modsulator.gemspec:19]: See https://github.com/sul-dlss/modsulator-app-rails/pull/245

**Review by mjgiarlo — APPROVED** [review]

**Comment by jcoyne** [comment]
This now requires a rebase.

**Comment by ndushay** [comment]
rebased and happy.

**Pull request merged**